### ISSUE TITLE
Documentation readf missing text.

### ISSUE THREAD

**Issue by HuskyNator** [Pure-D/code-d#338]
Documentation on readf is partially not displayed, however I do not know the intention of the actual documentation.
In source:
```
/**
 * Reads formatted data from `stdin` using $(REF formattedRead, std,_format).
 * Params:
 * format = The $(REF_ALTTEXT format string, formattedWrite, std, _format).
 * When passed as a compile-time argument, the string will be statically checked
 * against the argument types passed.
 * args = Items to be read.
 * Example:
----
// test.d
void main()
{
    import std.stdio;
    foreach (_; 0 .. 3)
    {
        int a;
        readf!" %d"(a);
        writeln(++a);
    }
}
----
$(CONSOLE
% echo "1 2 3" | rdmd test.d
2
3
4
)
 */
```

In VSCode however:
![image](https://user-images.githubusercontent.com/17416803/102703402-8deeb800-426e-11eb-94c9-bba6ce1fa40b.png)
![image](https://user-images.githubusercontent.com/17416803/102703392-6e578f80-426e-11eb-9267-6499c437d703.png)

**Comment by WebFreak001** [comment]
can you try if updating to serve-d beta or nightly fixes it? Should work there since this is already fixed.

**Comment by PetarKirov** [comment]
There was a typo in the ddoc comment of the function, which we recently fixed in the master branch:
https://github.com/dlang/phobos/pull/7732/files

@WebFreak001 if you're testing this on your side, make sure to try with phobos checkout from master.

@HuskyNator since the fix was merged in master, just a few hours after the 2.095.0 beta was branched off, the first dmd release to include this fix is going to be 2.096-beta1, [scheduled for 2021-02-15](https://dlang.org/changelog/release-schedule.html).

**Comment by HuskyNator** [comment]
Oh, hahaha, what ironic, this was fixed by a different issue i reported on issues.dlang.org, how ironic.
I'm surprised the two were related.

**Comment by HuskyNator** [comment]
By the way, looking at the code, the first part should have already been correct 'using .', or is that also not yet part of the current version?
![image](https://user-images.githubusercontent.com/17416803/102717576-8ae1df00-42e3-11eb-9159-999d46a8f0f1.png)

**Comment by HuskyNator** [comment]
I'm not sure how I would try this out with a beta version of serve-d, I simply used the marketplace, not very tech savy when it comes to vscode extensions or dependencies. 😅

**Comment by PetarKirov** [comment]
> Oh, hahaha, this was thus fixed by a different issue i reported on issues.dlang.org, how ironic.
I'm surprised the two were related. Thank you for your efforts!

Haha I am subscribed for notification to many D repos and since the titles of the issue were similar it caught my attention :smiley:  
I think the issues are indeed separate, even though related to the same function:
* The first line the ddoc text (``Reads formatted data from `stdin` using $(REF formattedRead, std,_format)``) was not parsed completely by the tool code-d is using internally.
* The `format` parameter docs had a typo - this is what was fixed by https://github.com/dlang/phobos/pull/7732

> I'm not sure how I would try this out with a beta version of serve-d, I simply used the marketplace, not very tech savy when it comes to vscode extensions or dependencies.

In VSCode you can change the settings of the whole editor and any extension that you have installed from the User Settings tab.

1. Open the VSCode User Settings (https://code.visualstudio.com/docs/getstarted/settings)

2. In the search bar type `d.servedReleaseChannel` and from the drop down select the nightly:

![image](https://user-images.githubusercontent.com/3475150/102718231-c4204c00-42ef-11eb-8dd5-2cacc63e264b.png)

**Comment by HuskyNator** [comment]
Ah webfreak mentioned the user settings. Woops skimmed over that part.
I'm unsure where to start since everything kept breaking horribly. . . so uhm . . . I'll give a summary in a bit.

**Comment by HuskyNator** [comment]
So enabling nightly / beta lead to . . . nothing, as in, errors and 'there are no suggestions'.
Strangely enough going back to stable doesn't work either (even with vscode restarts).
It might have something to do with the 4 'dcd-server.exe' processes still running after closing vscode, though this doesnt always happen, and closing them manually also doesn't fix it.
I've had some crazy issues after deinstalling the extension, which vscode eventually fixed after quite a few restarts;
![image](https://user-images.githubusercontent.com/17416803/102799469-dfa05b00-43b2-11eb-911d-6ee13d196eab.png)

Something may be wrong on my end, after reinstalling for a second time I'm also having issues with stable.
![image](https://user-images.githubusercontent.com/17416803/102815544-c86e6700-43cc-11eb-9359-805ab010508f.png)
Additionally, for some reason the extension keeps telling me there are too many subprojects, but seeing as there are 3, and the default setting should start warning after 4, this doesn't add up.

Tldr;
Honestly everything on my end is suddenly broken by a hilarious ammount, & I have no clue why 😂.
Sorry I was unable to confirm the issue was solved with never versions, now I just wish my old version ran without issues.

**Comment by WebFreak001** [comment]
@HuskyNator as you are on windows it might have issues with overriding versions. First make sure you have exactly one vscode window open. Also in your user settings disable "aggressive updates" because this will prompt you before downloading. Then change to nightly and reload the window. Now before updating, kill all dcd-server and serve-d exe processes to make sure nothing is running anymore and then confirm the update.

If that didn't work, delete the serve-d and DCD executables from `%APPDATA%/code-d`

**Comment by HuskyNator** [comment]
Ok, deleting %APPDATA%/code-d fixed reinstalls for the stable version. Shouldn't that be deleted when the extension is uninstalled? As can be seen in my case a broken version seemingly persists after reinstalls of the extension.
So when trying out nightly and beta now, I still don't get any suggestions (like for printf). It does however show using stable.
All using the following;
- changing the settings to stable/beta/nightly
- close vscode and running dcd-sverver/server-d processes (by the way I dont know how language 'servers' work, is there a reason they keep running after closing vscode?)
- remove the code-d folder and restart vscode
- download when promped (which is twice)
- close & reopen vscode and check whether readf yields suggestions using ctrl+space.

**Comment by HuskyNator** [comment]
I'm pretty sure it's not due to the files, but to be sure this is the folder I have tried this in (open folder in vscode);
[dag_3.zip](https://github.com/Pure-D/code-d/files/5726612/dag_3.zip)

**Comment by WebFreak001** [comment]
> Shouldn't that be deleted when the extension is uninstalled?
on windows there might be an issue that it can't delete the files because they are still running

I will try your zip

**Comment by HuskyNator** [comment]
Have not had any issues since, I'm assuming whatever was causing this is gone.
